### PR TITLE
Update 04-3-groundwork.html.md

### PR DESCRIPTION
**Pull request by Warkanlock**
A little changes over the documentation that would be good to have for future users:

1. Replace word **before** for **after**
2. Added new dependency over the MenuState.hx
3. Added dependency on the top of the `Main.hx`

### CODE REVIEW

File: documentation/01_tutorial/04-3-groundwork.html.md [hunk 1]
```diff
@@ -72,7 +73,12 @@ First up, let's create a simple menu. Eventually, we'll want a fancy `MenuState`
 
 	`screenCenter()` is a function which takes an object and centers it on the screen either horizontally, vertically or (by default) both. By calling it on our button, the button will be placed in the center of the screen.
 
-5. Next, we need to make sure that the game actually starts with our `MenuState`. Open `Main.hx` and replace the `PlayState` reference with `MenuState`.
+5. Next, we need to make sure that the game actually starts with our `MenuState`. Open `Main.hx` and replace the `PlayState` reference with `MenuState`. You have to add the reference to the MenuState file using on the top of the `Main.hx`
```
Comment (Gama11) [inline documentation/01_tutorial/04-3-groundwork.html.md:76]: No you don't, modules from the same package are imported automatically.

File: documentation/01_tutorial/04-3-groundwork.html.md [hunk 2]
```diff
@@ -37,12 +37,13 @@ First up, let's create a simple menu. Eventually, we'll want a fancy `MenuState`
 
 	As before with `FlxText`, the import should be added automatically. If not, you can also add missing imports via the light bulb that should appear after saving the file when you have your cursor over `FlxButton`.	
 
-2. Now, type the following on the lines before `super.create();` within the `create()` method:
+2. Now, type the following on the lines after `super.create();` within the `create()` method:
```
Comment (Gama11) [inline documentation/01_tutorial/04-3-groundwork.html.md:40]: Doesn't really matter.

File: documentation/01_tutorial/04-3-groundwork.html.md (hunk 1 above)
Comment (Warkanlock) [reply documentation/01_tutorial/04-3-groundwork.html.md]: Good to know 👍

File: documentation/01_tutorial/04-3-groundwork.html.md (hunk 2 above)
Comment (Warkanlock) [reply documentation/01_tutorial/04-3-groundwork.html.md]: I put both, after and before, is that ok?

**Pull request merged**